src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write your code here
+const thead = document.querySelectorAll('thead tr');
+const tbody = document.querySelectorAll('tbody tr');
+const tfoot = document.querySelectorAll('tfoot tr');
+
+const columnIndex = 1;
+
+thead.forEach((tr) => {
+  const th = tr.children[columnIndex];
+  const clonedTh = th.cloneNode(true);
+
+  tr.insertBefore(clonedTh, tr.children[columnIndex + 3]);
+});
+
+tbody.forEach((tr) => {
+  const th = tr.children[columnIndex];
+  const clonedTh = th.cloneNode(true);
+
+  tr.insertBefore(clonedTh, tr.children[columnIndex + 3]);
+});
+
+tfoot.forEach((tr) => {
+  const th = tr.children[columnIndex];
+  const clonedTh = th.cloneNode(true);
+
+  tr.insertBefore(clonedTh, tr.children[columnIndex + 3]);
```

Comment:
As with thead and tbody, verify that the tfoot rows have sufficient columns to handle the insertion at `columnIndex + 3`. This check is necessary to avoid runtime errors.